gen-apidocs/config/config.yaml

```diff
@@ -82,7 +82,7 @@ resource_categories:
       version: v1beta1
       group: storage
     - name: CSINode
-      version: v1
+      version: v1beta1
```

Comment:
no. CSINode is v1  GA now.

Reply:
OK. The change was made after the 1.17 release?

Reply:
Just checked: https://github.com/kubernetes/kubernetes/commit/9268d4053928488ea58829484bd59537b89da205#diff-ccf355e02c6054937bad28a1d8d754b4

msau42 committed on Oct 19, 2019

It is part of 1.17 release.